styles.c3c9715a.css

```diff
@@ -0,0 +1,2 @@
+a,abbr,acronym,address,applet,article,aside,audio,b,big,blockquote,body,canvas,caption,center,cite,code,dd,del,details,dfn,div,dl,dt,em,embed,fieldset,figcaption,figure,footer,form,h1,h2,h3,h4,h5,h6,header,hgroup,html,i,iframe,img,ins,kbd,label,legend,li,mark,menu,nav,object,ol,output,p,pre,q,ruby,s,samp,section,small,span,strike,strong,sub,summary,sup,table,tbody,td,tfoot,th,thead,time,tr,tt,u,ul,var,video{margin:0;padding:0;border:0;font-size:100%;font:inherit;vertical-align:baseline}article,aside,details,figcaption,figure,footer,header,hgroup,menu,nav,section{display:block}body{line-height:1}ol,ul{list-style:none}blockquote,q{quotes:none}blockquote:after,blockquote:before,q:after,q:before{content:"";content:none}table{border-collapse:collapse;border-spacing:0}.header{background-color:#fff;background-image:url(heder-background.5169347d.jpg);background-repeat:no-repeat;background-position:50%;background-size:cover;min-height:100vh;padding:0 20px 80px;box-sizing:border-box;grid-template-rows:1fr 1fr;display:grid;justify-content:center;grid-template-columns:repeat(2,130px);gap:20px}.header__title{align-self:end;grid-column:1/3;grid-row:2/3}.header__navigation{grid-row:1/2;grid-column:1/3}.title{font-family:Space Grotesk,sans-serif;font-weight:700;letter-spacing:-1.5px;text-align:center;color:#191919;font-size:36px;line-height:36px}.title__secondiary{font-size:32px;line-height:32px;letter-spacing:0}.logo__icon{width:125px;height:16px}.navigation{display:flex;justify-content:space-between;height:72px;align-items:center}.navigation__buttons{display:grid;align-items:center;grid-template-columns:24px 24px 24px}.burger{grid-column:3/4}.phone{display:grid}.phone__label{font-family:Inter;font-size:12px;font-weight:700;line-height:16px;letter-spacing:2px;position:absolute;color:#292929;transform:translateX(calc(-100% - 20px))}.phone:not(:hover) .phone__label{clip:rect(0 0 0 0);clip-path:inset(50%);height:1px;overflow:hidden;position:absolute;white-space:nowrap;width:1px}.main{padding:64px 20px;display:grid;justify-content:center;grid-template-columns:repeat(2,130px);gap:20px}.main__section,.main__title{grid-column:1/3}.section{display:grid;justify-content:center;grid-template-columns:repeat(2,130px);gap:20px}.section__picture{width:280px;height:280px;border-radius:4px}.section__title{font-family:Space Grotesk,sans-serif;font-size:18px;font-weight:700;line-height:25px;letter-spacing:0;color:#292929}.section__paragraph{font-family:Space Mono,sans-serif;font-weight:400}.section__paragraph,.section__price{font-size:16px;line-height:22px;letter-spacing:0;color:#292929}.section__price{font-family:Space Grotesk,sans-serif;font-weight:700}
```

Comment:
Consider using a CSS reset library like Normalize.css or a modern CSS reset to create a consistent base for your styling across different browsers.